Fix for https://qa.mandriva.com/show_bug.cgi?id=45393

cmnd field of struct scsi_cmnd changed after commit "[SCSI] Let
scsi_cmnd->cmnd use request->cmd buffer"
(64a87b244b9297667ca80264aab849a36f494884)

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>

---
 drivers/scsi/ppscsi.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/scsi/ppscsi.c
+++ b/drivers/scsi/ppscsi.c
@@ -1009,7 +1009,6 @@ int ppsc_biosparam (struct scsi_device *
 static int ppsc_inquire (PHA *pha, int target, char *buf)
 {
 	char inq[6] = {0x12,0,0,0,36,0};
-	int i;
 	struct scsi_cmnd cmd;
 	struct scsi_device dev;
 
@@ -1017,7 +1016,7 @@ static int ppsc_inquire (PHA *pha, int t
 	dev.id = target;
 	cmd.device = &dev;
 	cmd.cmd_len = 6;
-	for (i=0;i<6;i++) cmd.cmnd[i] = inq[i];
+	cmd.cmnd = inq;
 	cmd.sdb.table.nents = 0;
 	cmd.sdb.table.sgl = (void *) buf;
 	cmd.sdb.length = 36;
